apps/api/vitest.config.ts

```diff
@@ -0,0 +1,43 @@
+import tsconfigPaths from "vite-tsconfig-paths";
+import { defineConfig } from "vitest/config";
+
+import LogReporter from "./e2e/utils/log-reporter.js";
+
+export default defineConfig({
+  plugins: [tsconfigPaths()],
```

Comment:
This plugin allows Vitest to work with TS path aliases like we use in the project.